README.md

```diff
@@ -52,15 +52,18 @@ make pylint
 # Define a symbol from the command line - for example, setting the log level
 make test LIBRARY=ms-common DEFINE="LOG_LEVEL_VERBOSITY=LOG_LEVEL_WARN"
 
+# Intall python requirements for every project
+make install_requirements
+
+# Note that for python make commands, you will likely need to be in virtualenv. After running install_requirements, run
+source .venv/bin/activate
```

Comment:
Does this need to be run every time a new shell is opened, or does it work just running it once? If the first, we should add it to `~/.bashrc`, and if the second, we should just do it automatically in `make install_requirements`.

Reply:
I'll meet you halfway: we can append the virtual environment onto $(PATH) in the Makefile whenever a `make` command is run. This way it won't mess up other projects like CAN-Explorer or telemetry which have their own separate dependencies / virtualenvs.